packages/office-ui-fabric-react/src/components/Image/Image.test.tsx

```diff
@@ -15,126 +14,109 @@ const testImage1x1 = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYA
 const testImage1x2 = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAACCAYAAACZgbYnAAAAEklEQVQImWP4////fyYGBgYGAB32A/+PRyXoAAAAAElFTkSuQmCC';
 const testImage2x1 = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAABCAYAAAD0In+KAAAAEUlEQVQImWP8////fwYGBgYAGfgD/hEzDhoAAAAASUVORK5CYII=';
 const brokenImage = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgFcSJAAAAC0lEQVQImWP4DwQACfsD/eNV8pwAAAAASUVORK5CYII=';
-/* tslint:enable:no-unused-variable */
 
 describe('Image', () => {
+  beforeAll(() => {
+    // Manually set image height and width since there is no DOM
+    Object.defineProperty(HTMLImageElement.prototype, 'naturalHeight', { get: () => 1 });
+    Object.defineProperty(HTMLImageElement.prototype, 'naturalWidth', { get: () => 1 });
+  });
```

Comment:
Manually set `naturalHeight` and `naturalWidth` because jsdom defaults values to 0